src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const transformToNumber = (str) => {
+  return parseFloat(str.replace(/[$,]/g, ''));
+};
+const getEmployees = (list) => {
+  const result = list.map((employee) => {
+    return {
+      name: employee.textContent.trim(),
+      salary: transformToNumber(employee.getAttribute('data-salary')),
+    };
+  });
+
+  return SortList(result);
+};
+const SortList = (list) => {
+  return list.sort((a, b) => b.salary - a.salary);
+};
+
+const employeesArray = Array.from(document.querySelectorAll('ul li'));
+const employeesData = employeesArray.map((employee) => {
+  getEmployees(employee);
```

Comment:
There is an issue here: the `getEmployees` function is expected to process a list of employees, but you are passing a single employee element to it. You should pass the entire `employeesArray` to `getEmployees` instead.